cdap-watchdog/src/test/java/co/cask/cdap/metrics/collect/KafkaMetricsCollectionServiceTest.java

```diff
@@ -194,9 +194,8 @@ public void finished() {
       }
     });
 
-    Assert.assertTrue(semaphore.tryAcquire(expected.size(), 5, TimeUnit.SECONDS));
+    Assert.assertTrue(semaphore.tryAcquire(expected.size(), 15, TimeUnit.SECONDS));
 
-    Assert.assertEquals(expected.size(), metrics.size());
```

Comment:
Still figuring out this test.. not able to run due to missing some native libs on my machine.